README.md

```diff
@@ -167,6 +167,21 @@ If you have more than one key in `~/.ssh` you may need to specify which key `cf-
 $ export CF_REMOTE_SSH_KEY="~/.ssh/id_rsa.pub"
 ```
 
+### Working on the local host
+
+`cf-remote` can work on the local host when the target host is `localhost`. In this case, it executes commands locally without connecting over SSH.
```

Comment:
If you want to install, don't you need to be root? Maybe add that example as well?

`sudo cf-remote install -H localhost`

Reply:
It runs "sudo" automatically, sudo cf-remote shouldn't be needed.